apps/src/applab/designElements/slider.jsx

```diff
@@ -0,0 +1,162 @@
+/* global $ */
+
+var React = require('react');
```

Comment:
Please remove this require.  We've adopted an approach where our Rails app provides a single instance of React that is available globally, because we encountered issues with multiple React instances.  [Here's a related comment](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/StudioApp.js#L13).